web/src/layouts/settings/index.tsx

```diff
@@ -28,6 +29,9 @@ const Settings: React.FC = () => {
             <Tabs.Tab value="general" icon={<TbSettings size={20} />}>
               General
             </Tabs.Tab>
+            <Tabs.Tab value="characters" icon={<TbBriefcase size={20} />}>
```

Comment:
Why use the same icon as `Groups` tab does?

`TbUser` would probably be better.

Reply:
Missed this in the copy and paste mayhem. Fixed.